src/index.js

```diff
@@ -1,34 +1,87 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension, no-unused-vars */
-
 /* @jsx createElement */
 
+import { getDisplayNumbers } from './getDisplayNumbers';
+import { Operator } from './Operator';
+import { isNumeric } from './isNumeric';
+import { NumberCalculator } from './NumberCalculator';
+
 function createElement(tagName, props, ...children) {
   const element = document.createElement(tagName);
 
-  Object.entries(props || {}).forEach(([key, value]) => {
-    element[key.toLowerCase()] = value;
-  });
+  Object.entries(props || {})
+    .forEach(([key, value]) => {
+      element[key.toLowerCase()] = value;
+    });
 
-  children.flat().forEach((child) => {
-    if (child instanceof Node) {
-      element.appendChild(child);
-      return;
-    }
-    element.appendChild(document.createTextNode(child));
-  });
+  children.flat()
+    .forEach((child) => {
+      if (child instanceof Node) {
+        element.appendChild(child);
+        return;
+      }
+      element.appendChild(document.createTextNode(child));
+    });
 
   return element;
 }
 
-function render() {
-  const element = (
+function renderTemplate(element) {
+  const app = document.getElementById('app');
+  app.textContent = '';
+  app.appendChild(element);
+}
+
+const displayNumbers = getDisplayNumbers();
+const displayOperators = Operator.displays();
+
+function render(beforeNumber, beforeOperator, currentNumber, displayNumber = 0) {
+  /**
+   *
+   * @param firstNum: {number}
+   * @param operator: {string}
+   * @param secondNum: {number}
+   * @param clickNumber: {number}
+   */
+  function handleClickNumber(firstNum, operator, secondNum, clickNumber) {
+    const result = NumberCalculator.calculate(firstNum, operator, secondNum, clickNumber);
+    render(result.before, result.op, result.current, result.display);
+  }
```

Comment:
함수에 매개변수가 많아서 헷갈리는 것 같아요. 사실은 같은 값을 나타내는데, 다른 이름을 지으려다보니 다른 이름도 많이 나타난 것 같구요. 과감하게 생략하는 것도 좋을 것 같아요

```suggestion
  function handleClickNumber(number) {
    const result = NumberCalculator.calculate(beforeNumber, beforeOperator, currentNumber, number);
    render(result.before, result.op, result.current, result.display);
  }
```

Reply:
@hannut91 리뷰 감사합니다 :) 
내부 함수를 밖으로 빼고싶어서 최대한 render의 파라미터를 안쓰는 형태를 유지하고싶었는데,
그렇게 뺄게 아니라면 말씀하신대로 있는 파라미터 그대로 재사용하는게 나을것 같습니다 :)